code/modules/events/alien_infestation.dm

```diff
@@ -0,0 +1,33 @@
+
+/proc/alien_infestation(var/spawncount = 1)
+	var/list/vents = list()
+	var/success = FALSE
+	for(var/obj/machinery/atmospherics/unary/vent_pump/temp_vent in atmos_machines)
+		if(temp_vent.loc.z == map.zMainStation && !temp_vent.welded && temp_vent.network && temp_vent.canSpawnMice)
+			if(temp_vent.network.normal_members.len > 50) // Stops Aliens getting stuck in small networks. See: Security, Virology
+				vents += temp_vent
+
+	var/list/mob/dead/observer/candidates = get_active_candidates(ROLE_ALIEN, buffer=ALIEN_SELECT_AFK_BUFFER, poll="HEY KID, YOU WANNA BE AN ALIEN LARVA?")
```

Comment:
A little bit silly don't you think?

Reply:
Well yeah, but the content of that .dm is just code moved from the old event.dm that is otherwise deprecated now and I didn't want to change the legacy message. I don't think it's that big of a deal since the message is sent to observers too. (also it's only ever sent if an Admin triggers it manually so I think the cheekiness adds to it)

Reply:
Okay, it's fine if it's bus only I suppose